orion/1.12/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3'
+
+services:
+  orion:
+    image: fiware/orion:1.12.0
+    ports:
+      - "1026:1026"
+    command: -logLevel ${ORION_LOG_LEVEL} -dbhost ${MONGO_SERVICE_URI} -rplSet ${REPLICASET_NAME} -dbTimeout 10000
```

Comment:
missing default values of env variables.

Reply:
@taliaga The Service Manager hasn't had the little parsing problem fixed yet, so keeping them off for now. Will put them all back in once that's solved